cmd/kubeadm/app/phases/token/bootstrap_test.go

```diff
@@ -47,6 +47,22 @@ preferences: {}
 users:
 - name: kubernetes-admin`
 
+const testIPv6Config = `apiVersion: v1
```

Comment:
Agree with @pmichali. Please parameterize the `server:` field here and add both good and bad cases.